### PR TITLE
Update Homebrew installation docs for InfluxDB 1.8 and 2.0

### LINKED ISSUE #2381: Update macOS install instructions
The recommended way to install InfluxDB on macOS is with Homebrew. We held off adding Homebrew install instructions in the 2.0 docs because the brew package was still 1.x. That's no longer the case. Homebrew now ships 2.0.

We need to update the macOS install instructions in both 2.0 and 1.x.

##### Use brew to install 2.0
```sh
brew install influxdb
```

##### Use brew to install 1.x
```sh
brew install influxdb@1
```

I don't know if we want to go into this, but you can install both 1.x and 2.0 with:

```sh
brew install influxdb@2 influxdb@1
```

##### Relevant URLs
- https://docs.influxdata.com/influxdb/v2.0/get-started/
- https://docs.influxdata.com/influxdb/v1.8/introduction/install/?t=macOS

### PR DESCRIPTION
**Pull request by pierwill**
Closes #2381.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/get-started.md [hunk 1]
```diff
@@ -28,9 +28,17 @@ _See [Differences between InfluxDB Cloud and InfluxDB OSS](#differences-between-
...
-### Download and install InfluxDB v2.0
+### Install InfluxDB v2.0
+
+We recommend installing InfluxDB v2.0 using [Homebrew](https://brew.sh/) on MacOS.
+With Homebrew installed, run:
+
+```sh
+brew update
+brew install influxdb
+```
```
Comment (sanderson) [inline content/influxdb/v2.0/get-started.md:39]:
~~~suggestion
We recommend using [Homebrew](https://brew.sh/) to install InfluxDB v2.0 on macOS:

```sh
brew update
brew install influxdb
```
~~~

File: content/influxdb/v2.0/get-started.md [hunk 2]
```diff
@@ -28,9 +28,17 @@ _See [Differences between InfluxDB Cloud and InfluxDB OSS](#differences-between-
 
 <!-------------------------------- BEGIN macOS -------------------------------->
 {{% tab-content %}}
-### Download and install InfluxDB v2.0
+### Install InfluxDB v2.0
```
Comment (sanderson) [inline content/influxdb/v2.0/get-started.md:31]: I think we need to add some content sections under this heading:

- Use Homebrew
- Manually download and install

All of the existing instructions only apply to manually downloading and installing. Then added anchored links to each.

**Review by sanderson — CHANGES_REQUESTED** [review]
I think we need to add some sections under

File: content/influxdb/v1.8/introduction/install.md [hunk 3]
```diff
@@ -194,32 +194,41 @@ To have InfluxDB start at system boot, add `influxd_enable="YES"` to `/etc/rc.co
...
+brew services start influxdb@1
 ```
 
-And then to start InfluxDB now, run:
+Or, if you don't want/need launchctl, in a separate terminal window you can just run:
 
 ```bash
-launchctl load ~/Library/LaunchAgents/homebrew.mxcl.influxdb.plist
+influxd -config /usr/local/etc/influxdb.conf
 ```
```
[the remark this answers is not in the document]
Comment (pierwill) [reply content/influxdb/v1.8/introduction/install.md:214]: On second thought, I'll remove this stuff. It is pretty much pulled verbatim from what Homebrew tells users once they've installed the package.

**Review by sanderson — APPROVED** [review]
Ship it!

**Pull request merged**